.github/dependabot.yml

```diff
@@ -10,4 +10,8 @@ updates:
     schedule:
       interval: "daily"
     ignore:
-      - dependency-name: "org.apache.maven.plugins:maven-compiler-plugin"
\ No newline at end of file
+      - dependency-name: "org.apache.maven.plugins:maven-compiler-plugin"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
I think it's fair keeping it `daily`.

Reply:
Since we use major versions at the actions only, we should get minor updates anyway. So IMHO the bot should just raise an issue once a major update is available. Thus I think a ``weekly`` interval should be fine. But but ultimately you decide ...